test/test2/bbdgyg

```diff
@@ -0,0 +1,2 @@
+dummy file
```

Comment:
What is this file used for?

Reply:
This file is under test2 directory which can only be accessed by the owner and not any other group or user. This directory I used to test the fs.is_writable function, that returns false if the passed in a directory that cannot be accessed.